applications/ecallmgr/src/ecallmgr_fs_conferences.erl

```diff
@@ -319,9 +319,8 @@ init([]) ->
 %% @end
 %%--------------------------------------------------------------------
 -spec handle_call(any(), pid_ref(), state()) -> handle_call_ret_state(state()).
-handle_call({'conference_create', Props, Node}, _, State) ->
-    lager:debug("created conference ~p", [Props]),
-    Conference = conference_from_props(Props, Node),
+handle_call({'conference_create', JObj, Node}, _, State) ->
```

Comment:
The spec for `create/2` asks for a `kz_proplist()` but you process it as a `kz_json:object()` here. Can you update the spec for `create/2`?